Source/WebGPU/WebGPU/CommandEncoder.swift

```diff
@@ -25,7 +25,8 @@
 
 import Metal
 import WebGPU_Internal
-
+public typealias WTFString = String
```

Comment:
could this be `public typealias WTFString = WTF.String` just for clarity, or that wouldn't work?

Reply:
That is a good suggestion. On me to make that change in the next one.